jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/Dispatcher.java

```diff
@@ -51,22 +51,22 @@ public class Dispatcher implements RequestDispatcher
 
     private final ContextHandler _contextHandler;
     private final HttpURI _uri;
-    private final String _pathInContext;
+    private final String _uriInContext;
     private final String _named;
 
-    public Dispatcher(ContextHandler contextHandler, HttpURI uri, String pathInContext)
+    public Dispatcher(ContextHandler contextHandler, HttpURI uri, String uriInContext)
```

Comment:
Can we remove the `uriInContext` parameter and just derive it from the `contextHandler` and `HttpURI`? That way it's clear what we're doing - it's not really clear if the `uriInContext` is encoded or decoded.

Reply:
ditto....